src/rpc/client/ts/src/server.ts

```diff
@@ -80,22 +60,25 @@ export async function startServer(
     rootPath: rootPath,
     port: port,
   })
-  const artifact = new Artifact('omega-edit-grpc-server', version, rootPath)
 
-  let args = [`--interface=${host}`, `--port=${port}`]
+  const pidfile = getPidFile(rootPath, port)
```

Comment:
The pidFile must be configurable by the caller, so it needs to come in as an argument.  `getPidFile()` is fine to put into the test fixture, but this is code that the integrator will actually call, so we need to leave it up to the caller to decide.

Reply:
For example in VS Code, the PID files will go into that App data directory.